lib/super_diff/configuration.rb

```diff
@@ -56,5 +60,29 @@ def add_extra_inspector_classes(*classes)
       :add_extra_inspector_class,
       :add_extra_inspector_classes,
     )
+
+    def set_alpha_color(color)
+      @alpha_color = color
+    end
+    alias_method(
```

Comment:
Since this is the first time we're introducing this, I'm wondering if it makes sense to have two names for this. It feels like it'd be less to keep track of if we stuck to one name. What do you think about removing the aliases?